navody/asus.html

```diff
@@ -0,0 +1,15 @@
+<h3 id="asus">Asus</h3>
```

Comment:
Chceme mít návody v Gitu? Nechal bych je jenom v OneSky, ať je jasný zdroj pravdy.

Reply:
FAQ stejně tak

Reply:
Pak web nepůjde vybuildit offline/lokálně bez přístupu k mému API secretu..

Reply:
Nešlo by tam dát jenom nějaké dummy soubory kde bude napsáno "Edit this file in OneSky"?

Reply:
OK.